### PR TITLE
Add express session maxAge

### PR DESCRIPTION
**Pull request by bgavrilMS**
This is to force AAD to re-evaluate the user access, since the cache storage is done on a per user basis, i.e. shared by several sessions that the user could open.

### CODE REVIEW

File: samples/msal-node-samples/auth-code-distributed-cache/src/app.ts [hunk 1]
```diff
@@ -6,27 +6,34 @@
 import fs from "fs";
```
Comment (bgavrilMS) [inline samples/msal-node-samples/auth-code-distributed-cache/src/app.ts:6]: Only actual change is at line 81, the rest is linting.

**Comment by bgavrilMS** [comment]
@Robbie-Microsoft - please review this.

**Review by Robbie-Microsoft — APPROVED** [review]

**Comment by bgavrilMS** [comment]
@trwalke - can you please have a quick look at this?

**Comment by bgavrilMS** [comment]
@pmaytak - can I please get a PR review for this?

**Review by sameerag — APPROVED** [review]

**Review by pmaytak — APPROVED** [review]

**Pull request merged**